test/functions.js

```diff
@@ -173,6 +173,13 @@
     assert.notStrictEqual(myObj, void 0, 'object is created if second argument used as key');
     assert.strictEqual(myObj, myObjAlias, 'object is cached if second argument used as key');
     assert.strictEqual(myObj.value, 'a', 'object is not modified if second argument used as key');
+
+    // test symbol
+    if (typeof Symbol !== 'undefined') {
+      var itSelf = _.memoize(function(self){return self;});
+      var sym = Symbol();
+      assert.strictEqual(sym, itSelf(sym), 'symbol can be memoized, too');
+    }
```

Comment:
This is not enough. Firstly, it invokes the function only once, so the lookup behavior is never triggered. Secondly, it doesn't check that distinct symbols can actually be distinguished from each other.